build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -8,5 +8,8 @@
 # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 #flowExportInterval: 60s
 
-# Provide the transport protocol for the flow aggregator collecting process, which is tcp or udp.
-#aggregatorTransportProtocol: "tcp"
+# Provide the transport protocol for the flow aggregator collecting process, which is tls, tcp or udp.
+#aggregatorTransportProtocol: "tls"
+
+# Provide DNS name of flow aggregator for generating TLS certificate. It must match the flowCollectorAddr parameter in the antrea-agent config.
+#flowAggregatorDNSName: "flow-aggregator.flow-aggregator.svc"
```

Comment:
Could you add an ending newline to avoid the warning?

Reply:
Sure. Thanks!